index.html

```diff
@@ -76,6 +76,8 @@ <h3 class="title">Tonic</h3>
             </div>
           </div>
         </div>
+    </div>
+    </section>
         
 	<section class="about">
```

Comment:
For some reason, there are some sections I am unable to highlight that need some changing. I will however do so by use of screenshots for support.

- [ ] Kindly make the "Lets connect" to use a ```<hx>``` instead of a ```<span>``` element as it is currently.

**This is the part I am talking of:**

![image](https://user-images.githubusercontent.com/86318284/185758865-0ee2ee14-b765-4847-8db0-b0d5566fc77b.png)

- [ ] Kindly align your "About me" section making use of the correct margins and padding s as provided in the chosen Figma design

**This is how I am seeing your design right now:**

![image](https://user-images.githubusercontent.com/86318284/185758957-b8cac1f9-b92b-4351-84e9-ee74d17ea5ff.png)


**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185758983-99ba074b-c309-429f-accf-4b8986d123d0.png)

- [ ] It seems like you did not complete the whole "About me section" There is the "skills" section which is part of the about me section that is currently missing. Kindly include it.

**This is the missing section as per the provided Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185759035-b069d8f1-f2dc-4c05-89e1-233dc95d2bdb.png)